### PR TITLE
Translate pytorch_vision_resnext.md

### PR DESCRIPTION
**Pull request by joonkeekim**
## 라이선스 동의
_변경해주시는 내용에 BSD 3항 라이선스가 적용됨을 동의해주셔야 합니다._<br />
_더 자세한 내용은 [기여하기 문서](https://github.com/9bow/PyTorch-tutorials-kr/blob/master/CONTRIBUTING.md)를 참고해주세요._<br />
_동의하시면 아래 `[ ]`를 `[x]`로 만들어주세요._<br />

- [x] 기여하기 문서를 확인하였으며, 본 PR 내용에 BSD 3항 라이선스가 적용됨에 동의합니다.

## 관련 이슈 번호
_이 Pull Request와 관련있는 이슈 번호를 적어주세요._<br />
_이슈 또는 PR 번호 앞에 #을 붙이시면 제목을 바로 확인하실 수 있습니다. (예. #999 )_

- **이슈 번호**: #14 (숫자)

## PR 종류
_이 PR에 해당되는 종류 앞의 `[ ]`을 `[x]`로 변경해주세요._<br />
- [ ] 오탈자를 수정하거나 번역을 개선하는 기여
- [x] 번역되지 않은 튜토리얼을 번역하는 기여
- [ ] 공식 튜토리얼 내용을 반영하는 기여
- [ ] 위 종류에 포함되지 않는 기여

## PR 설명
_이 PR로 무엇이 달라지는지 대략적으로 알려주세요._

pytorch_vision_resnext.md 문서를 번역하였습니다.

### CODE REVIEW

File: pytorch_vision_resnext.md [hunk 1]
```diff
@@ -86,10 +87,10 @@ for i in range(top5_prob.size(0)):
 
 ### Model Description
```
Comment (des00) [inline pytorch_vision_resnext.md:88]: 여기랑 아래의 Resource도 번역하는게 어떨까 싶습니다.
기존에 번역 올려두신 분도 이쪽 부분 다 번역하셨더라고요.
https://github.com/9bow/PyTorch-hub-kr/pull/12/files

File: pytorch_vision_resnext.md [hunk 2]
```diff
@@ -24,23 +24,24 @@ model = torch.hub.load('pytorch/vision:v0.9.0', 'resnext50_32x4d', pretrained=Tr
 model.eval()
 ```
 
-All pre-trained models expect input images normalized in the same way,
-i.e. mini-batches of 3-channel RGB images of shape `(3 x H x W)`, where `H` and `W` are expected to be at least `224`.
-The images have to be loaded in to a range of `[0, 1]` and then normalized using `mean = [0.485, 0.456, 0.406]`
-and `std = [0.229, 0.224, 0.225]`.
+사전 훈련된 모든 모델들은 입력 이미지들이 같은 방식으로 정규화된것으로 생각합니다.
```
Comment (Taeyoung96) [inline pytorch_vision_resnext.md:27]: 맞춤법상,  
'정규화 된 것으로'라고 바꿔야 할 것 같습니다!

File: pytorch_vision_resnext.md [hunk 3]
```diff
@@ -24,23 +24,24 @@ model = torch.hub.load('pytorch/vision:v0.9.0', 'resnext50_32x4d', pretrained=Tr
 model.eval()
 ```
 
-All pre-trained models expect input images normalized in the same way,
-i.e. mini-batches of 3-channel RGB images of shape `(3 x H x W)`, where `H` and `W` are expected to be at least `224`.
-The images have to be loaded in to a range of `[0, 1]` and then normalized using `mean = [0.485, 0.456, 0.406]`
-and `std = [0.229, 0.224, 0.225]`.
+사전 훈련된 모든 모델들은 입력 이미지들이 같은 방식으로 정규화된것으로 생각합니다. 
+즉, `(3 x H x W)` 모양의 3채널 RGB 이미지를 말하고 `H` 와 `W`은 각각 최소 `224` 이상을 기대합니다.
+이미지는 `[0, 1]` 범위로 로드한 다음 `mean = [0.485, 0.456, 0.406]` 과 `std = [0.229, 0.224, 0.225]`
```
Comment (Taeyoung96) [inline pytorch_vision_resnext.md:29]: 이미지는 `[0, 1]` 범위 로드한 다음 >> 이미지는 `[0,1]` 범위로 불러온 다음  
이라고 바꾸는 건 혹시 어떨까요?

**Review by Taeyoung96 — COMMENTED** [review]
고생하셨습니다!

**Comment by Taeyoung96** [comment]
@joonkeekim   
안녕하세요.  Feedback에 대한 추가적인 수정 및 comment를 남겨주시면 감사하겠습니다!

**Review by 9bow — CHANGES_REQUESTED** [review]
@joonkeekim 님, 위 @Taeyoung96 님의 덧글 확인 부탁드립니다. :)

**Comment by hyoyoung** [comment]
마지막 피드백 이후 15일이 지나면 보통 닫게 됩니다.
진행이 많이 된 PR이므로, 마무리 피드백 처리 부탁드리겠습니다

**Comment by joonkeekim** [comment]
확인이 늦어 매우 늦은 피드백 죄송합니다..! 우선적으로 말씀해주신 1. 용어 2. 추가 번역 해결하고 커밋 올렸습니다.  
하지만 이미 번역되고 Merge된 pytorch_vision_resnext.md가 있어 PR을 닫아도 괜찮을까요?

**Comment by hyoyoung** [comment]
그럼 본 PR은 닫겠습니다. 수고 많으셨습니다
